docker/README.md

```diff
@@ -1,71 +1,76 @@
-# Deploying the 1Password SCIM bridge using Docker
+# Deploy 1Password SCIM Bridge with Docker
 
-This example describes the methods of deploying the 1Password SCIM bridge using Docker. The Docker Compose and Docker Swarm managers are available and deployment using each manager is described below.
+*Learn how to deploy 1Password SCIM Bridge using Docker Compose or Docker Swarm.*
 
-## Preparing
+**Table of contents:**
 
-Ensure you've read through the [PREPARATION.md](/PREPARATION.md) document before beginning deployment.
```

Comment:
It might be nice to standardize this line across the deployment options. AWS says:

"Before beginning, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there."